### DISCUSSION TITLE
progress_bar_color in config/livewire is not working

### DISCUSSION THREAD

**Discussion by MouhamadHamadani** [livewire/livewire#7011] [1. Help]
In the documentation:
[https://livewire.laravel.com/docs/navigate#customizing-the-progress-bar] In this section it tells you, you can change the progress bar color in the config/livewire.php,
I opened the required file and scrolled down to the navigate section but there was no progress_bar_color found,
I added it but nothing changed, would anyone be able to help me with it?! thank you

**Comment by PerryvanderMeer** [comment]
That option is already included in the documentation, but has not actually been released yet. Will be included in the next release.

**Reply by MouhamadHamadani** [reply]
oh okay thank you

**Reply by marcorieser** [reply]
Seems like it's released in [`v3.0.6`](https://github.com/livewire/livewire/releases/tag/v3.0.6)

**Reply by MouhamadHamadani** [reply]
I have livewire 3.0.8 and it's not working

**Reply by gunnery34** [reply]
Update to Livewire 3.0.10 it's work
I've tried it!

**Reply by MouhamadHamadani** [reply]
I updated to 3.0.10 and it didn't work, can you please help with it?

**Reply by gunnery34** [reply]
run command `php artisan config:clear` and `php artisan view:clear`
after you run, refresh your browser and try again

**Reply by MouhamadHamadani** [reply]
still didn't work

**Comment by joshhanley** [comment]
We're in the process of investigating all wire:navigate issues and I have tested out your scenario. I believe this has been fixed so I'm going to close this. If you're still experiencing problems, please open a new discussion. Thanks!
